tests/ttnn/unit_tests/operations/eltwise/test_composite.py

```diff
@@ -235,6 +239,25 @@ def test_unary_composite_hardsigmoid_ttnn(input_shapes, device):
     assert comp_pass
 
 
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
```

Comment:
For consistency, maybe we can use the same format and skip `torch.Size` part?

```@pytest.mark.parametrize(
    "input_shape",
    [
        [32, 32],
        [5, 96, 64],
    ],
)
```

Reply:
@ayerofieiev-tt yes, done.